ecs-cli/modules/compose/cli/ecs/app/factory_test.go

```diff
@@ -31,10 +32,27 @@ func TestPopulateContext(t *testing.T) {
 	globalContext := cli.NewContext(nil, globalSet, nil)
 	cliContext := cli.NewContext(nil, nil, globalContext)
 	ecsContext := &ecscompose.Context{}
+
+	// Create a temprorary directory for the dummy ecs config
+	tempDirName, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatal("Error while creating the dummy ecs config directory")
+	}
+
 	os.Setenv("AWS_REGION", "us-east-1")
+	os.Setenv("AWS_ACCESS_KEY", "AKIDEXAMPLE")
+	os.Setenv("AWS_SECRET_KEY", "secret")
+	os.Setenv("HOME", tempDirName)
+	defer func() {
+		os.Unsetenv("AWS_REGION")
+		os.Unsetenv("AWS_ACCESS_KEY")
+		os.Unsetenv("AWS_SECRET_KEY")
+		os.Unsetenv("HOME")
+		os.Remove(tempDirName)
```

Comment:
Can you move this closer to the line where tempDirName is created? The same comment also applies to the 'TestNewDefaultDestination' method.